MetaMask/metamask-mobile
# feat: add notifications UI views

## **Description**

This is the last PR from a series of 6 PRs to implement the new notifications feature on MetaMask. This brings all the backbones and UI elements together to present our users:

- OptIn Screen
- Settings Screen
- Notifications List Screen
- Notification Details Screen

## **Related issues**

Fixes:

## **Manual testing steps**


1. Go to this Settings screen, open the new Notifications option and enable it.
2. Go to main screen, click on the bell icon
3. See the empty screens.
4. Perform any onChain transaction
5. Click on the Notification and see its details

## **Screenshots/Recordings**



https://github.com/MetaMask/metamask-mobile/assets/44679989/80029408-ea8d-424e-9f1d-735423d5ff02

### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
